spec/factories/users.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :user do
-    client_slug { ENV['CLIENT_SLUG_DEFAULT'] || 'ncr' }
+    client_slug { ENV['CLIENT_SLUG_DEFAULT'] || nil }
```

Comment:
maybe we should just set to `nil` rather than using env var? I haven't ever set it, so not sure what the use case for env var is.